### PR TITLE
Add "New" svg icon for showing where people can look at new categories or features

### PR DESCRIPTION
**Pull request by TreTuna**
_Description of changes:_
In working with the new Geo category, @hdworld11 presented an ask to add this `New` icon on the Category Card and on the Sidebar navigation to better show developers where our newest features and categories are.

The implementation is quick and open to any feedback on that side of things. I mainly wanted to see how this would work and get a POC up and going here for discussion.

image
image

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Milan-Shah — APPROVED** [review]

**Comment by jakeburden** [comment]
Staging URL for testing / QA: https://geo-add-new-svg.d1oosfztpocl9c.amplifyapp.com/

**Comment by jakeburden** [comment]
Thanks @TreTuna! I left one comment for review.

File: src/components/Card/styles.tsx [hunk 1]
```diff
@@ -136,6 +136,9 @@ export const CardDetail = styled.div`
 
   > h4 {
     color: var(--font-color);
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
```
[the remark this answers is not in the document]
Comment (TreTuna) [reply src/components/Card/styles.tsx]: Reviewed with @jakeburden in Slack. Was decided to remove `align-items` from the Card styles, but not the directory.

**Comment by mauerbac** [comment]
Spoke to Tre. Closing

**Pull request closed without merging**